### PR TITLE
fix login issues

### PR DESCRIPTION
**Pull request by Tolu-Mals**
# Summary
Fixes issue with login button not responding sometimes

### Notes
If you're currently experiencing issues with Metamask, you need to disconnect directly from Metamask first
Screenshot 2024-04-02 at 10 13 59 PM

closes https://github.com/EveripediaNetwork/issues/issues/2473

### CODE REVIEW

**Comment by Adebesin-Cell** [comment]
@Tolu-Mals I think the wagmi wrapper works for it!

**Comment by Adebesin-Cell** [comment]
> @Tolu-Mals I think the wagmi wrapper works for it!

https://github.com/EveripediaNetwork/ep-ui/assets/66301634/39581bbd-aac2-423a-8cab-4751aadcbcfa

**Comment by Tolu-Mals** [comment]
> > @Tolu-Mals I think the wagmi wrapper works for it!
> 
>  Screen.Recording.2024-04-02.at.4.47.20.PM.mov

Oh, I tested earlier and it didn't seem to work. Let me try again.

**Comment by Adebesin-Cell** [comment]
> > > @Tolu-Mals I think the wagmi wrapper works for it!
> > 
> > 
> > Screen.Recording.2024-04-02.at.4.47.20.PM.mov
> 
> Oh, I tested earlier and it didn't seem to work. Let me try again.

Oh, use the deployment link, it works well.

**Comment by Tolu-Mals** [comment]
> > > > @Tolu-Mals I think the wagmi wrapper works for it!
> > > 
> > > 
> > > Screen.Recording.2024-04-02.at.4.47.20.PM.mov
> > 
> > 
> > Oh, I tested earlier and it didn't seem to work. Let me try again.
> 
> Oh, use the deployment link, it works well.

deployment link doesn't seem to be working

**Comment by Adebesin-Cell** [comment]
> > > > > @Tolu-Mals I think the wagmi wrapper works for it!
> > > > 
> > > > 
> > > > Screen.Recording.2024-04-02.at.4.47.20.PM.mov
> > > 
> > > 
> > > Oh, I tested earlier and it didn't seem to work. Let me try again.
> > 
> > 
> > Oh, use the deployment link, it works well.
> 
> deployment link doesn't seem to be working

I mean this https://iq-wiki-4oo034m0y-prediqt.vercel.app/

**Comment by Tolu-Mals** [comment]
> > > > > > @Tolu-Mals I think the wagmi wrapper works for it!
> > > > > 
> > > > > 
> > > > > Screen.Recording.2024-04-02.at.4.47.20.PM.mov
> > > > 
> > > > 
> > > > Oh, I tested earlier and it didn't seem to work. Let me try again.
> > > 
> > > 
> > > Oh, use the deployment link, it works well.
> > 
> > 
> > deployment link doesn't seem to be working
> 
> I mean this https://iq-wiki-4oo034m0y-prediqt.vercel.app/

doesn't work on my end

**Comment by Adebesin-Cell** [comment]
> > > > > > > @Tolu-Mals I think the wagmi wrapper works for it!
> > > > > > 
> > > > > > 
> > > > > > Screen.Recording.2024-04-02.at.4.47.20.PM.mov
> > > > > 
> > > > > 
> > > > > Oh, I tested earlier and it didn't seem to work. Let me try again.
> > > > 
> > > > 
> > > > Oh, use the deployment link, it works well.
> > > 
> > > 
> > > deployment link doesn't seem to be working
> > 
> > 
> > I mean this [iq-wiki-4oo034m0y-prediqt.vercel.app](https://iq-wiki-4oo034m0y-prediqt.vercel.app/)
> 
> doesn't work on my end

hm, weird.

**Review by Adebesin-Cell — APPROVED** [review]
LGTM 🚀
